src/frontend/components/property-type/currency/filter.tsx

```diff
@@ -0,0 +1,30 @@
+import { CurrencyInput, CurrencyInputProps, FormGroup } from '@adminjs/design-system'
+import React, { FC, useCallback } from 'react'
+import { EditPropertyProps } from '../base-property-props'
+import { PropertyLabel } from '../utils/property-label'
+
+type CurrencyEditPropertyProps = EditPropertyProps & CurrencyInputProps
+
+const Filter: FC<CurrencyEditPropertyProps> = (props) => {
+  const { onChange, property, filter } = props
+
+  const handleChange = useCallback((value) => {
+    onChange(property.path, value)
+  }, [])
```

Comment:
This doesn't need memoizing - `CurrencyInput` doesn't use `React.memo` so it will rerender anyway.

Reply:
Removed